src/main/java/com/synopsys/integration/alert/channel/jira/server/web/JiraServerCustomEndpoint.java

```diff
@@ -79,9 +81,17 @@ public JiraServerCustomEndpoint(JiraServerChannelKey jiraChannelKey, CustomEndpo
             PluginManagerService jiraAppService = jiraServicesFactory.createPluginManagerService();
             String username = jiraProperties.getUsername();
             String password = jiraProperties.getPassword();
-            Response response = jiraAppService.installMarketplaceServerApp(JiraConstants.JIRA_APP_KEY, username, password);
-            if (BooleanUtils.isTrue(response.isStatusCodeError())) {
-                return Optional.of(responseFactory.createBadRequestResponse("", "The Jira server responded with error code: " + response.getStatusCode()));
+            try {
+                Response response = jiraAppService.installMarketplaceServerApp(JiraConstants.JIRA_APP_KEY, username, password);
+                if (BooleanUtils.isTrue(response.isStatusCodeError())) {
+                    return Optional.of(responseFactory.createBadRequestResponse("", "The Jira server responded with error code: " + response.getStatusCode()));
+                }
+            } catch (IntegrationRestException e) {
+                if (RestConstants.NOT_FOUND_404 == e.getHttpStatusCode()) {
+                    return Optional.of(responseFactory.createNotFoundResponse(
+                        "The marketplace listing of the Alert Issue Property Indexer app may not support your version of Jira. Please install the app yourself or request a compatibility update. " + e.getMessage()));
```

Comment:
Can we actually change this part of the message "Please install the app yourself or request a compatibility update." to "Please install the app manually or request a compatibility update."